*** a/salt/utils/network.py	Tue Mar 22 13:27:49 2016
--- b/salt/utils/network.py	Tue Mar 22 13:27:54 2016
***************
*** 590,596 ****
                      if not salt.utils.is_sunos():
                          ipv6scope = mmask6.group(3) or mmask6.group(4)
                          addr_obj['scope'] = ipv6scope.lower() if ipv6scope is not None else ipv6scope
!                 if addr_obj['address'] != '::' and addr_obj['prefixlen'] != 0:  # SunOS sometimes has ::/0 as inet6 addr when using addrconf
                      data['inet6'].append(addr_obj)
          data['up'] = updown
          if iface in ret:
--- 590,599 ----
                      if not salt.utils.is_sunos():
                          ipv6scope = mmask6.group(3) or mmask6.group(4)
                          addr_obj['scope'] = ipv6scope.lower() if ipv6scope is not None else ipv6scope
!                 if salt.utils.is_sunos():
!                     if addr_obj['address'] != '::' and addr_obj['prefixlen'] != 0:  # SunOS sometimes has ::/0 as inet6 addr when using addrconf
!                         data['inet6'].append(addr_obj)
!                 else:
                      data['inet6'].append(addr_obj)
          data['up'] = updown
          if iface in ret:
